canonical/maas-ui
# feat(kvm): update KVM tables to fetch via the new API

## Done

- Update the LXD VM tables to fetch the machines via the new API.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Karura](/HACKING.md#karura)
- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Connect your UI to bolla.
- Go to KVM and click on the cluster then click on the Polong host.
- You should see one machine in the table.
- You should be able to sort and search the table (note that the loading skeleton will be added in a followup: https://github.com/canonical/app-tribe/issues/1323).

## Fixes

Fixes: canonical/app-tribe#1117.
